src/Command/ProductExportCommand.php

```diff
@@ -60,14 +62,22 @@ class ProductExportCommand extends Command implements ContainerAwareInterface
     /** @var EntityRepositoryInterface */
     private $channelRepository;
 
+    /** @var EntityRepositoryInterface */
+    private $currencyRepository;
+
+    /** @var ExportSettings */
+    private $exportFilters;
+
     public function __construct(
         SalesChannelService $channelService,
         FeedFactory $feedFactory,
         PushImportService $pushImportService,
         UploadService $uploadService,
         Traversable $productFields,
         EntityRepositoryInterface $languageRepository,
-        EntityRepositoryInterface $channelRepository
+        EntityRepositoryInterface $channelRepository,
+        EntityRepositoryInterface $currencyRepository,
+        ExportSettings $exportFilters
```

Comment:
can we change this to `$exportSettings` since the class name has been changed too?